plan/policy.go

```diff
@@ -31,6 +32,31 @@ var Policies = map[string]Policy{
 	"upsert-only": &UpsertOnlyPolicy{},
 }
 
+type OwnershipPolicy struct {
+	Owner string
+}
+
+func (p *OwnershipPolicy) Apply(changes *Changes) *Changes {
+	changes.Create = filterByOwner(p.Owner, changes.Create)
```

Comment:
as u correctly mentioned below, create will not have ownership information, therefore can be passed further